roles/ceph-facts/tasks/facts.yml

```diff
@@ -22,7 +22,7 @@
 
 - name: set_fact docker_exec_cmd
   set_fact:
-    docker_exec_cmd: "docker exec ceph-mon-{{ hostvars[groups[mon_group_name][0]]['ansible_hostname'] }}"
+    docker_exec_cmd: "{{ container_binary }} exec ceph-mon-{{ hostvars[groups[mon_group_name][0]]['ansible_hostname'] }}"
```

Comment:
is it really required ? because we never call the ceph-facts role with container_binary set to podman